catalogs/catalog-hive/src/main/java/org/apache/gravitino/catalog/hive/HiveTable.java

```diff
@@ -76,55 +72,15 @@ private HiveTable() {}
    * @return A new HiveTable instance Builder.
    */
   public static HiveTable.Builder fromHiveTable(Table table) {
-    // Get audit info from Hive's Table object. Because Hive's table doesn't store last modifier
-    // and last modified time, we only get creator and create time from Hive's table.
-    AuditInfo.Builder auditInfoBuilder = AuditInfo.builder();
-    Optional.ofNullable(table.getOwner()).ifPresent(auditInfoBuilder::withCreator);
-    if (table.isSetCreateTime()) {
-      auditInfoBuilder.withCreateTime(Instant.ofEpochSecond(table.getCreateTime()));
-    }
+    AuditInfo auditInfo = ConvertUtil.getAuditInfo(table);
 
-    StorageDescriptor sd = table.getSd();
-    Distribution distribution = Distributions.NONE;
-    if (sd.getBucketCols() != null && !sd.getBucketCols().isEmpty()) {
-      // Hive table use hash strategy as bucketing strategy
-      distribution =
-          Distributions.hash(
-              sd.getNumBuckets(),
-              sd.getBucketCols().stream().map(NamedReference::field).toArray(Expression[]::new));
-    }
+    Distribution distribution = ConvertUtil.getDistribution(table);
```

Comment:
`CovertUtil` seems not so reasonable, maybe "HiveUtil" or "HiveTableUtil", "HiveTableConverter"?

Reply:
renamed to `HiveTableConverter`